views/partials/top-level/security-tips.hbs

```diff
@@ -25,49 +20,52 @@
   </div>
 </main>
 <section class="container bg-light articles">
-  <div class="row jst-cntr">
+  <div class="row">
     <div class="col-9">
+      <section class="st-intro-wrap">
+        {{#each (getSecurityTipsIntro)}}
+          <p class="st-intro">{{{ this }}}</p>
+        {{/each}}
+      </section>
+
       {{#each (getArticleCopy)}}
-        <div id="{{ class }}"class="article-wrapper">
+        <div id="{{ class }}" class="st-{{ class }} article-wrapper">
+          <div class="ico {{ class }}">{{> (lookup . 'pathToPartial')}}</div>
+          <span class="security-tip-subhead">{{subhead}}</span>
           <h2 class="article-headline {{ class }}">{{ title }}</h2>
           {{#each copy}}
 
-            {{#if lead}}
-              <p class="article-paragraph lead">{{ lead }}</p>
-            {{/if}}
+            {{#each leads}}
+              <p class="article-paragraph lead">
+                <span class="drop-cap">
+                  {{ ./../dropCap }}
+                </span>
+                <span>{{ this }}</span>
+              </p>
+            {{/each}}
 
-            {{#if subhead}}
-              <h3 class="article-subhead">{{ subhead }}</h3>
+            {{#if toggles}}
+              {{#each this.toggles}}
+                <div class="toggle-parent inactive st-toggle-wrapper">
+                  <div class="toggle-info-wrapper">
+                    <button class="toggle svg-wrap article-toggle" aria-label="{{ getString "show-content" }}">
+                      <span class="article-subhead toggle-subhead">{{{ subhead }}}</span>                    
```

Comment:
Again on this `{{{ }}}` I double-checked that `subhead` has no user-generated-content.